misc/config_tools/schema/config.xsd

```diff
@@ -166,7 +166,7 @@
     </xs:element>
     <xs:element name="MAX_PCI_DEV_NUM" default="96">
       <xs:annotation acrn:title="Maximum number of PCI devices" acrn:views="advanced">
-        <xs:documentation>Specify the static memory allocation of maximum number of PCI devices. (Default value is 96)</xs:documentation>
+        <xs:documentation>Specify the maximum number of PCI devices. The default value is calculated from the board configuration file. If you have PCI devices that were not detected by the Board Inspector, you may need to change this maximum value.</xs:documentation>
```

Comment:
We should mention this impacts the amount of memory used to maintain information about these PCI devices.